### PR TITLE
add possibility to define snmp timeout/retries

### PR DESCRIPTION
**Pull request by to-kn**
since we have one nas with responds sometimes rather slowly, we had the need to raise the timeouts

### CODE REVIEW

File: check_synology.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 parser.add_argument("-w", help="warning value for selected mode", type=int)
 parser.add_argument("-c", help="critical value for selected mode", type=int)
 parser.add_argument("-p", help="the snmp port", type=int, dest="port", default=161)
+parser.add_argument("--timeout", help="timeout for snmp connection", type=int, default=10)
+parser.add_argument("--retries", help="retries for snmp connection if timeout occurs", type=int, default=3)
```
Comment (wernerfred) [inline check_synology.py:23]:
```suggestion
parser.add_argument("-t", help="timeout for snmp connection", type=int, default=10)
parser.add_argument("-r", help="retries for snmp connection if timeout occurs", type=int, default=3)
```
i would favor the short argument version

File: check_synology.py [hunk 2]
```diff
@@ -29,6 +31,8 @@
 mode = args.mode
 warning = args.w
 critical = args.c
+snmp_timeout = args.timeout
+snmp_retries = args.retries
```
Comment (wernerfred) [inline check_synology.py:35]:
```suggestion
snmp_timeout = args.t
snmp_retries = args.r
```

**Review by wernerfred — CHANGES_REQUESTED** [review]
LGTM 👍🏻 
thx for your contribution. Only minor change requests to use the short argument version and please add the new options also to the [usage section](https://github.com/wernerfred/check_synology#usage) in the `README.md`

File: icinga2/synology-command.conf [hunk 3]
```diff
@@ -55,6 +55,15 @@ object CheckCommand "check_synology" {
...
+    "--timeout" = {
+       order = 7
+       set_if = "$synology_snmp_timeout$"
+       value = "$synology_snmp_timeout$"
+    }
+    "--retries" = {
+       order = 7
+       set_if = "$synology_snmp_retries$"
+       value = "$synology_snmp_retries$"
+    }
```
Comment (wernerfred) [inline icinga2/synology-command.conf:67]:
```suggestion
    "--t" = {
       order = 7
       set_if = "$synology_snmp_timeout$"
       value = "$synology_snmp_timeout$"
    }
    "--r" = {
       order = 7
       set_if = "$synology_snmp_retries$"
       value = "$synology_snmp_retries$"
    }
```

**Comment by to-kn** [comment]
@wernerfred like that?

**Review by wernerfred — APPROVED** [review]

**Comment by wernerfred** [comment]
@all-contributors add @to-kn for code

**Pull request merged**